modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/WalkthroughSiteLevel.testcase

```diff
@@ -60,4 +66,69 @@ definition {
 		}
 	}
 
+	@description = "LPS-159844. When a walkthrough step is defined and a step is below the fold, user can see popover is."
+	@priority = "3"
+	test CanRenderWhenStepIsBelowFold {
+		property portal.acceptance = "true";
+
+		task ("Given walkthrough enabled site-level") {
+			Walkthrough.setupWalkthroughPage(
+				filename = "walkthrough_configuration_multiple_page.json",
+				siteName = "Site Name",
+				widgetList = "Blogs,JS Walkthrough Sample");
```

Comment:
Let's just leave this setupWalkthroughPage alone and not complicate it with other widgets.

You can just add another task prior to adding JS walkthrough sample ("Given vertically long ckeditor sample portlet") and use frontend ckeditor sample portlet. Maybe add Lorem ipsum text if need to extend the portlet down further.

Reply:
Doesn't look like this change was made

Reply:
I kept the macro as it was, because I need the sample of the Walkthrough sample to be added after ckeditor sample to stay below and the scroll to be used. That is, add the page > add the ckeditor sample > add the Walkthrough sample.

Reply:
The `JSONLayout.addWidgetToPublicLayout `macro adds the widget below if it is already on the page